packages/api/src/mappings/oura/activity.ts

```diff
@@ -144,7 +144,7 @@ export const ouraDailyActivityResponse = z.object({
   total_calories: z.number(),
   day: z.string(),
   timestamp: z.string(),
-});
+}).optional();
```

Comment:
should we make this nullish just in case?

Reply:
All 3 of these are the first item in an array. So it should only be either undefined or the object